alf/examples/muzero_atari_conf.py

```diff
@@ -427,30 +587,42 @@ def _scale_grad(scale):
     max_unroll_length=5,
     learn_policy_temperature=1.0)
 
+
+def model_ctor(*args, **kwargs):
+    model = SimpleMCTSModel(*args, **kwargs)
+    # Nero optimizer normalizes the parameters. We need to do this in the beginning
+    # so that the target_model is same as the model.
+    if optimizer_type == 'NERO':
+        NeroPlus.initialize(model)
+    return model
+
+
 alf.config(
     "MuzeroRepresentationImpl",
-    model_ctor=SimpleMCTSModel,
-    # The following line can be commented out if GPU memory is large enough
-    reanalyze_batch_size=1280 if use_small_net else 640,
+    model_ctor=model_ctor,
+    data_augmenter=data_augmenter,
     num_unroll_steps=5,
     td_steps=10,
     reanalyze_algorithm_ctor=MCTSAlgorithm,
+    random_action_after_episode_end=True,
     reanalyze_td_steps_func=  #LinearMaxAgeTdStepFunc(),
     LinearTdStepFunc(max_bootstrap_age=1.2, min_td_steps=1),
     train_repr_prediction=train_repr_prediction,
     train_game_over_function=train_game_over_function,
     reanalyze_ratio=1.0,
-    target_update_period=400,
+    target_update_period=100,
     target_update_tau=1.0)
 
 alf.config(
     "MuzeroAlgorithm",
     discount=discount,
     enable_amp=True,
-    representation_learner_ctor=MuzeroRepresentationImpl,
-    mcts_algorithm_ctor=MCTSAlgorithm,
+    mcts_algorithm_ctor=partial(
+        MCTSAlgorithm, num_parallel_sims=8, num_simulations=200),
```

Comment:
Would like to confirm it is intended to use a different set of parameter for rollout (as opposite to reanalyze), so that we search more thoroughly at rollout?

Reply:
Yes